GiniSwitchSDK/Example/GiniTariffSDK/SdkBuilder.swift

```diff
@@ -30,4 +35,20 @@ struct SdkBuilder {
         
         return sdk
     }
+    
+    static func clientCredentials() -> (clientId:String, clientSecret:String, clientDomain:String) {
+        if let path = Bundle.main.path(forResource: "Credentials", ofType: "plist"),
+            let keys = NSDictionary(contentsOfFile: path),
+            let clientId = keys[self.clientIDKey] as? String,
+            let clientSecret = keys[self.clientSecretKey] as? String,
+            let clientDomain = keys[self.clientDomainKey] as? String,
+            !clientId.isEmpty,
+            !clientSecret.isEmpty,
+            !clientDomain.isEmpty {
+            return (clientId, clientSecret, clientDomain)
+        }
```

Comment:
That's a lot of conditions, can you refactor them somehow? Maybe combine the client* initialisations and checking into some methods or so.

Reply:
Actually, the check for emptiness (`clientSecret.isEmpty`) is redundant. We check that again in `ExtractionsManager` before authenticating. I'll remove it and we will have 5 checks in the if. And since they are quite straight-forward, I'd keep them as-is.